Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -340,7 +343,13 @@ void EnhancementsWidget::AddDescriptions()
       QT_TR_NOOP("Disables the blending of adjacent rows when copying the EFB. This is known in "
                  "some games as \"deflickering\" or \"smoothing\". Disabling the filter has no "
                  "effect on performance, but may result in a sharper image, and causes few "
-                 "graphical issues.\n\n\nIf unsure, leave this checked.");
+                 "graphical issues.\n\nIf unsure, leave this checked.");
+  static const char TR_ARBITRARY_MIPMAP_DETECTION_DESCRIPTION[] = QT_TR_NOOP(
+      "Enables detection of arbitrary mipmaps, which some games use for special "
+      "distance-based effects.\nSometimes causes textures to appear blurry at increased "
```

Comment:
Here's a small recommendation here for legibility:

>May have false positives that result in blurry textures at high internal resolution, such as in games that use very low resolution mipmaps.

Reply:
Yeah that is more legible, but I'll keep "increased" rather than "high", as it doesn't necessarily have to be high IR to be visible. I can tell a difference even at 2x.